### DISCUSSION TITLE
"strict mode" for arbitrary values

### DISCUSSION THREAD

**Discussion by bryanjhickey** [tailwindlabs/tailwindcss#11286] [Help] [answered]
In a[ previous version of the docs](https://v2.tailwindcss.com/docs/just-in-time-mode#arbitrary-value-support) there was mention of a possible future feature "strict mode" for Arbitrary values to deny use of arbitrary values. 

Has this been progressed or released? Can't seem to find any mention of it the current docs or in the repo?

**Comment by adamwathan** [comment, marked as answer]
Nah probably not going to build that, in practice arbitrary values have been really useful for basically every project we’ve worked on so I think this is better handled by code review/team standards.
